core/src/main/res/values/strings.xml

```diff
@@ -229,6 +229,8 @@
     <string name="download_type_media">Media file</string>
     <string name="download_type_image">Image</string>
     <string name="download_request_error_dialog_message_prefix">An error occurred when trying to download the file:\u0020</string>
+    <string name="null_value_error_dialog_message_prefix">Error</string>
+    <string name="null_value_error_error">No podcast was provided that could be shown.</string>
```

Comment:
The name of this string should be something more specific to the actual usage. I suggest `error_show_null_podcast`